src/main/java/com/scalar/cassy/command/AbstractCommand.java

```diff
@@ -33,6 +34,13 @@
       description = "A data directory to take backups from or restore backup to")
   protected String dataDir;
 
+  @CommandLine.Option(
+      names = {"--store-type"},
+      required = true,
+      paramLabel = "STORE_TYPE",
+      description = "The store type [AWS_S3, AZURE_STORAGE_BLOB, FILE_SYSTEM]")
```

Comment:
Cassy : back to the future :)
```suggestion
      description = "The store type [AWS_S3, AZURE_STORAGE_BLOB]")
```

Reply:
Oops :)